### PR TITLE
sniffer v0.5.1 (new formula)

### PR DESCRIPTION
**Pull request by chenjiandongx**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/sniffer.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "go" => :build
+
+  uses_from_macos "libpcap"
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    system bin/"sniffer", "-l"
```
Comment (SMillerDev) [inline Formula/sniffer.rb:17]: Can we check for some output here?

Comment (chenjiandongx) [reply Formula/sniffer.rb]: Sure, see the latest commit.

File: Formula/sniffer.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  uses_from_macos "libpcap"
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    output = shell_output("#{bin}/sniffer -l")
+    assert_equal true, (output.include? "lo")
```
Comment (SMillerDev) [inline Formula/sniffer.rb:18]:
```suggestion
    assert_match "lo", output
```

Comment (chenjiandongx) [reply Formula/sniffer.rb:18]: got it

Comment (chenjiandongx) [reply Formula/sniffer.rb]: update sniffer.rb already

**Comment by chenjiandongx** [comment]
@SMillerDev any other suggestions?

File: Formula/sniffer.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  uses_from_macos "libpcap"
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    output = shell_output("#{bin}/sniffer -l")
+    assert_match "lo", output
```
Comment (SMillerDev) [inline Formula/sniffer.rb:18]: Might as well simplify this more
```suggestion
    assert_match "lo", shell_output("#{bin}/sniffer -l")
```

Comment (chenjiandongx) [reply Formula/sniffer.rb]: done

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenjiandongx** [comment]
ping @chenrui333 , this PR looks ready to be merged.

**Review by SMillerDev — APPROVED** [review]
Thanks @chenjiandongx  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**